desc/objectives/linear_objectives.py

```diff
@@ -670,6 +670,10 @@ def build(self, eq, use_jit=True, verbose=1):
             Level of output.
 
         """
+        if eq.iota is None:
+            raise RuntimeError(
+                "Attempt to fix rotational transform on an equilibrium with no profile assigned"
```

Comment:
* with no rotational transform profile

this error could be thrown on an eq with a current profile, so this would make it more clear